src/Hyperwallet.js

```diff
@@ -90,7 +90,7 @@ export default class Hyperwallet {
      * @param {api-callback} callback - The callback for this call
      */
     listUsers(options, callback) {
-        const LIST_USER_FILTERS = ["clientUserId", "email", "programToken", "status", "verificationStatus",
+        const LIST_USER_FILTERS = ["clientUserId", "email", "programToken", "status", "verificationStatus","taxVerificationStatus",
```

Comment:
Will this change also handle createUser, retrieveUser and updateUser calls?

Reply:
Not for this SDK, as those methods just use a data JSON and there is no USER model for node sdk. The addition of the filter list is so that list calls can be organized or filtered for a certain value type